plugins/chat/app/services/chat/create_message.rb

```diff
@@ -44,7 +44,7 @@ class CreateMessage
       step :process_direct_message_channel
     end
     step :publish_new_thread
-    step :publish_new_message_events
+    step :process
```

Comment:
I sneaked in this little change related to previous PRs, I think it now makes more sense to call it process